packages/ember-glimmer/lib/utils/process-args.js

```diff
@@ -1,10 +1,11 @@
+import { assert } from 'ember-debug';
 import { symbol } from 'ember-utils';
 import { ARGS } from '../component';
 import { UPDATE } from './references';
 import { MUTABLE_CELL } from 'ember-views';
 import { ACTION } from '../helpers/action';
 
-// ComponentArgs takes EvaluatedNamedArgs and converts them into the
+// processComponentArgs takes CapturedNamedArguments and converts them into the
 // inputs needed by CurlyComponents (attrs and props, with mutable
 // cells, etc).
```

Comment:
Should this be a YUI doc comment?

Reply:
Probably not, it is more of an "implementors comment" than something written for Ember user's...